### PR TITLE
Bug/fixes

### PR DESCRIPTION
**Pull request by adborden**
Some random fixes before the PEC meeting.

### CODE REVIEW

File: app/components/appMainModule/committeePageModule/committeePageRoutes.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ function committeePageRoutes($stateProvider) {
     template: require('./committeePage.html'),
     ncyBreadcrumb: {
       label: '{{ committee.name }}',
-      parent: 'appMain.locality'
+      parent: 'appMain'
```
Comment (KyleW) [inline app/components/appMainModule/committeePageModule/committeePageRoutes.js]: Interesting. So this fixes the breadcrumbs in one way, but breaks it in another. Before this change, the breadcrumbs looked correct if you navigated to the page through the site, but were blank if you directly loaded the url. Now, there's a breadcrumb if you load the page directly, but I think it's broken if you navigate to the page through the site? Or that could be all wrong. Breadcrumbs, huh.

File: app/components/appMainModule/committeePageModule/committeePage.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-<div class="fluid-container">
+<div class="container">
```
Comment (adborden) [inline app/components/appMainModule/committeePageModule/committeePage.html]: On my larger screen the mocks are in a narrow container, but turns out that's just invision adding it's own container and being confusing. @elinaru can you clarify what the container widths should be? Fixed containers in bootstrap are up to `1170px` on the largest desktop screens. The other option would be to have a fluid/full-width container.

File: app/components/appMainModule/committeePageModule/committeePageRoutes.js (hunk 1 above)
Comment (adborden) [inline app/components/appMainModule/committeePageModule/committeePageRoutes.js]: Yeah, I don't even think `appMain.locality` is a real state anymore (we're using `appMain.localePage`). You're right, I'm stumped on why the breadcrumbs break now.

File: app/components/appMainModule/committeePageModule/committeePage.html (hunk 2 above)
Comment (adborden) [inline app/components/appMainModule/committeePageModule/committeePage.html]: nm, I'm thinking of a conversation I had with @elinaru about the container on the homepage. The mock shows the contributor list wider than the footer, so let's change this to `container-fluid`.

**Pull request merged**